docs/how-to-configure-new-repository.md

```diff
@@ -23,7 +23,12 @@ for the repository.
    `foo-approvers` and `foo-triagers` explicitly, with the "Maintainer" GitHub 
    privileges. This allows repository maintainers to invite new approvers and 
    triagers to the team.
-3. The team `foo-triagers` has `Triage` permissions for the repository.
+3. The team `foo-triagers` has `Triage` permissions for the repository. If repository
+   is using the Project Boards, `foo-triagers` should have `Write` permissions to
+   have access to the Project Boards. Note, by not adding members of `foo-triagers` to
+   CODEOWNERS file, repository restricts triagers from counting a triagers approvals for
+   PRs. See the [Policies](#policies) section that suggest to ensure that `Require
+   review from Code Owners` is checked.
```

Comment:
I think their approvals will still count for the minimum number of required approvals (by members with write access) and the code owner check will pass even if only one single code owner has approved.

Reply:
yes, it still will be green. But at least some level of distinction between two roles may give some confidence to maintainers. Do you think some rewording is needed here? Or an additional note? 

I hope that GitHub can change this =)

Reply:
I just wanted to point this out in case it was missed but if there's no way to work around this restriction I think it's fine.